### PR TITLE
all done pagination is left

### PR DESCRIPTION
**Pull request by adarsh7523**

### CODE REVIEW

File: src/app/dashboard/speciality/FormInModal.tsx [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+          }}
+        >
+          <Typography variant="h4" gutterBottom>
+            {title} Speciality
+          </Typography>
+        </Box>
+        <Formik
+          initialValues={formValues}
+          enableReinitialize //to reinitialize the form when it gets stored values from backend
+          // validationSchema={userValidation}
```
Comment (coder-faraz) [inline src/app/dashboard/speciality/FormInModal.tsx]: where is the validation? using Yup

File: src/app/dashboard/speciality/page.tsx [hunk 2]
```diff
@@ -1,67 +1,118 @@
...
+    (state: RootState) => state.speciality
+  );
 
+  // Fetch data based on currentPage and pageSize
   const { value: data, refetch } = useGetSpeciality(
-    {} as Speciality,
-    '/speciality/get-specialization',
-     '672c681f76ab84e9f25f0539',
+    null,
+    "Speciality/get-Speciality",
```
Comment (coder-faraz) [inline src/app/dashboard/speciality/page.tsx]: make the url as lower case

**Review by coder-faraz — CHANGES_REQUESTED** [review]
left comments

File: src/app/dashboard/speciality/FormInModal.tsx (hunk 1 above)
Comment (adarsh7523) [reply src/app/dashboard/speciality/FormInModal.tsx]: done

File: src/app/dashboard/speciality/page.tsx (hunk 2 above)
Comment (adarsh7523) [reply src/app/dashboard/speciality/page.tsx]: done

**Pull request merged**